backend/clubs/views.py

```diff
@@ -1276,6 +1277,49 @@ def upload_file(self, request, *args, **kwargs):
 
         return file_upload_endpoint_helper(request, code=club.code)
 
+    @action(detail=True, methods=["get"])
+    def history(self, request, *args, **kwargs):
```

Comment:
Do we need to add this route to the `ClubPermission` permission class? If not, I think the default is only members & above can see routes that aren't explicitly handled.